sdk/durabletask/ci.mgmt.yml

```diff
@@ -1,7 +1,6 @@
 # NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
 
 trigger: none
-
```

Comment:
@raych1 Why is the tool making changes to other files not related to `carbonoptimization` ? Is this a known thing?

Reply:
Previously we had a [known bug](https://github.com/Azure/azure-sdk-tools/issues/9646) but was fixed. It seems to be similar issue not only for management plane script. 
I created [issue#10540](https://github.com/Azure/azure-sdk-tools/issues/10540) for tracking.
CC: @live1206